sway/swaynag.c

```diff
@@ -21,6 +22,41 @@ static void handle_swaynag_client_destroy(struct wl_listener *listener,
 	swaynag->client = NULL;
 }
 
+struct wl_client * swaynag_spawn_detailed(char *cmd[], struct swaynag_instance *swaynag) {
```

Comment:
This can be static

Style: no space on the right of `*`

Reply:
Hm, even in return type? Is not it unreadable?
I've fixed it but probably you can consider leaving space there

Reply:
No. Adding a space makes it unreadable.

More seriously, the style guide is the style guide. Fighting the style guide isn't worth your time.

Reply:
No offence. I've fixed that as you've pointed to missing space.
Just tried to clarify